build/hygiene.js

```diff
@@ -195,7 +195,7 @@ function createGitIndexVinyls(paths) {
 				}
 
 				cp.exec(
-					`git show :${relativePath}`,
+					`git show :"${relativePath}"`,
```

Comment:
Looks like VS Code has a slightly different fix, would suggest using this one instead so we keep in sync with their stuff

https://github.com/microsoft/vscode/blob/main/build/hygiene.js#L234

Reply:
That line still throws the same error, since the windows path isn't quoted.

Reply:
I'm curious why they don't have this same issue then...

At the very least let's copy over what they have and add the extra quotes around the windows one.

Reply:
That javascript regex file is the only spaced file I've seen, so maybe they don't change it that often.